cgi/top_translators.pl

```diff
@@ -63,6 +63,12 @@
 				if (results.errors.length === 0) {
 					dataSource = dataSource.concat(results.data);
 				}
+				else {
+					for (var i = 0; i < results.errors.length; ++i) {
```

Comment:
I would use a forEach:
```js
results.errors.forEach(error => {
  console.warn("...");
}
```

Reply:
There's no huge advantage over the `for` loop, and arrow functions don't work in a few browsers. I'll leave it as-is for now.